microsoft/axe-windows
# chore: Remove duplicate signing warning from build

#### Describe the change
The signed build is generating this warning during the build phase:

[warning]E:\A\_work\12\a\MicroBuild\Plugins\MicroBuild.Plugins.Signing.1.1.293\build\MicroBuild.Plugins.Signing.targets(93,5): **Warning : File E:\A\_work\12\s\src\CLI_Installer\bin\Release\AxeWindowsCLI.msi is listed more than once in the file list. Duplicate instances are being ignored**

This is now the only warning that our build is producing, so it would be good to remove it and "get clean". After some investigation, it appears that the MSI is added to the signing list once by `src\CLI_Installer\CLI_Installer.wixproj` and once by `build\NetStandardRelease.targets`

Since the .targets file is how we sign all of our other projects, it seemed logical to allow it to sign the MSI file, as well, and to remove the lines that were causing the duplicate signing warning.

Validation:
- Create a signed release (https://dev.azure.com/mseng/1ES/_build/results?buildId=12758278&view=results) from a branch with this change
- Confirm that it builds cleanly (no warnings) and that the signing validation step is happy
- Download the MSI file from the artifacts
- Download the MSI file from the artifacts of a pipeline build without the change (https://dev.azure.com/mseng/1ES/_build/results?buildId=12757078&view=results)
- Compare the MSI files:
  - File sizes are identical
  - Primary certificate is identical (same serial number and other properties)
  - Time stamp certificate is different (expected)

Validated by building a signed release, ensuring that it didn't generate the warning, downloading the MSI file from the artifacts, and comparing its signature to an MSI file from a build loop that was gener


> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.
